pandas/core/base.py

```diff
@@ -990,7 +990,7 @@ def factorize(self, sort=False, na_sentinel=-1):
         from pandas.core.algorithms import factorize
         return factorize(self, sort=sort, na_sentinel=na_sentinel)
 
-    def searchsorted(self, key, side='left'):
+    def searchsorted(self, key, side='left', sorter=None):
```

Comment:
actually I would do this like we do ``_validate_kwargs``, e.g. add ``**kwargs`` to the signature and accept ``sorter``, but raise on other args. The reason is we don't actually use this arg (and therefore are not documenting it). you *may* want/need to move ``_validate_kwargs`` to ``core.common`` (or similar)